data/org.gnome.shell.gschema.xml.in

```diff
@@ -18,6 +18,15 @@
         using the Alt-F2 dialog.
       </description>
     </key>
+    <key name="enable-app-center" type="b">
```

Comment:
Well, the effect will be the same as not implementing this. So I confirmed none of our repos is using it and assumed this is something we may use for demos and other cases, as the original task mentioned.